### DISCUSSION TITLE
TimeOut on Discord channel, no idea why

### DISCUSSION THREAD

**Discussion by AlphaQUp** [Zer0-bit/gaggiuino#536] [Q&A]
Hi everyone. I asked one question in the Discord channel and looks like a bot put me on TimeOut for a week. I have no idea what  the cause is, to be honest (although I am new to Discord and still learning my way around). Does anyone know how to get out of that state? I need to confirm something before ordering some parts. Thanks in advance, cheers!

**Comment by banoz** [comment]
did you try to ping everyone?

**Reply by cptbucky** [reply]
This has just happened to me too, it looks like discord doesn't give you a reason. I could be wrong, at least searching my profile says nothing.

I was timed out for 7 days as I hit send on a message that had a video attached, not an issue prior so presumably not that. I did use the @ to ping 2 people to say thank you for their help, could it be that?

Also would be great to know how I get it removed, can I DM someone specifically?!

**Reply by Zer0-bit** [reply]
If you read the rules again you'll understand why the timeout, also it usually sends the reason via a private message but if you have your pm blocking incoming comms you won't be notified for obv reasons

**Reply by cptbucky** [reply]
Apologies, the 2nd bullet in the rules clearly states "do NOT tag people" this is my mistake.
